_invokers/command.md

```diff
@@ -12,4 +12,121 @@ releases: [
 ]
 ---
 
-<!-- TODO write something meaningful -->
+The command function invoker provides a Docker base layer for a function consisting of a single command.
+It accepts gRPC requests, invokes the command for each request in the input stream,
+and sends the command's output to the stream of gRPC responses.
+
+## Install as a riff invoker
```

Comment:
Installation should be handled by the invoker layout. Including it again will lead to confusion.

Reply:
Actually i think limiting each page to a single markdown content blob is simpler and allows us to put more important things at the top (just like the README on github). I'd like to to work on the content first, with examples, and then decide if we want to formalize the installation bits in the template.